internal/stack/boot.go

```diff
@@ -13,11 +13,20 @@ import (
 	"github.com/elastic/elastic-package/internal/builder"
 	"github.com/elastic/elastic-package/internal/configuration/locations"
 	"github.com/elastic/elastic-package/internal/files"
+	"github.com/elastic/elastic-package/internal/profile"
 )
 
-// DockerComposeProjectName is the name of the Docker Compose project used to boot up
+// baseComposeProjectName is the base name of the Docker Compose project used to boot up
 // Elastic Stack containers.
-const DockerComposeProjectName = "elastic-package-stack"
+const baseComposeProjectName = "elastic-package-stack"
+
+// DockerComposeProjectName returns the docker compose project name for a given profile.
+func DockerComposeProjectName(profile *profile.Profile) string {
+	if profile.ProfileName == "default" {
+		return baseComposeProjectName
+	}
```

Comment:
If this condition is added, the resulting container names would be the same as previously keeping backwards compatibility.